### ISSUE TITLE
Weight calculation assumes a hardcoded constant 80kg

### ISSUE THREAD

**Issue by kubark42** [iltis42/XCVario#243]
**Describe the bug**
 When I add a new glider to the polar, I do not expect it to calculate a weight for me. I prefer to include this information myself, as referenced from the published POH.

However, XCVario calculates the weight by multiplying the wing-loading used during the polar tests, the wing area, and an 80kg offset. https://github.com/iltis42/XCVario/blob/cb9faa019a2baaf9bd6c61f8aef775203d02c482/main/S2F.cpp#L38C22-L38C36

For a SWAG (scientific wild ass guess), an 80kg pilot isn't bad, but it gets into a bit of trouble with TMGs, as there it's not fair to assume polar coordinates come from an empty plane with a single parachute-wearing pilot.

This violates the principle of least surprise, as the user cannot see where the weight is coming from, and it's not clear either when looking in Polars.cpp.

**Expected behavior**
Weight should either not be automatically calculated (i.e. by including weight into the polars) or the UI should state clearly how the weight value is calculated.

**Criticality**

Not at all critical. If the maintainers disagree, that's okay, this Issue leaves breadcrumbs for future people who come across this. It would, however, still ideally be documented in the UI. It took quite a few minutes of searching around to find the line of code where weight was calculated.

**Comment by iltis42** [comment]
The mentioned code calculates the "empty weight" by this (yes agree SWAG) assumption for the pilot who did the polar reference flight, if the user didn't enter the correct value. 
The empty weight has nothing to do with the actual pilot. Its related to the reference polar and the wingload used at that time. 
Polars out of FHB were just given with their wingload or total weight, very rarely with a crew weight, so the info about the pilot is and so empty weight simply not avail. See polar of Duo-Discus below as an example. 
But even it was there, this won't help much, its another glider. The actual empty weight of your concrete glider may differ from the glider prototype where the polar have has been evaluated as (even its the same type) your gliders is usually be heavier from GFK/CFK hygroscopic in humid environment, repainting, repairs or extensions. So its a good idea to look into the last weight and balance sheet and take the exact empty weight from there. As you see in the above code, this calculation is only done once nothing is entered and the value is zero. 
The weight of the pilot is not a SWAG, its for sure regarded from "Crew Weight", and regarded during calculation of ballasted polar in the same manner than ballast is taken into account.

![image](https://github.com/iltis42/XCVario/assets/34288016/e178ddea-49cd-466f-ada7-e9cb8f56cd60)

**Comment by kubark42** [comment]
Agreed. It's a decent SWAG and you've got to put *something* there. And it's better to put something realistic than 0.

That being said, if it had been documented instead of silent then it would have saved me some time. When I created my own glider profile for `Polars.cpp`, It surprised me to see an empty weight which was not correct for my glider, but was not far off. Out of perhaps an overabundance of caution, I decided to dive into the code to understand if I had done something wrong. It turns out that not, that instead there is a calculation which provides a SWAG to get things going somewhat close.

> this calculation is only done once nothing is entered and the value is zero.

 My experience is slightly different. From what I've seen calculation happens any time the glider's named profile is changed, i.e. going from    a `LS 8` to `DG 100`. The normal user wouldn't be changing glider types, though, so this is definitely a niche case.

**Comment by iltis42** [comment]
Yes, changing the glider, this recalculation is mandatory of course. 
Agree to add a hint in the manual and clarify the pilots weight to be a fixed value of 80 kg. 

![image](https://github.com/iltis42/XCVario/assets/34288016/563c3814-1181-4cb1-a626-817391e1b577)

**Comment by kubark42** [comment]
Thanks!

BTW, I didn't know that composites absorbed net weight over time. We've seen in boats how water can enter the matrix osmotically, forming blisters, but that's traditionally something which happens when the boat is in contact with liquid water. So it was really great that you included that information in the manual. (As unbelievable as it is for these days, I read the whole thing cover to cover.)

**Comment by iltis42** [comment]
Its an issue in very humid environment, e.g. if the trailor has some leak and water comes in. I came across this when repainting gliders and measured residual torque in alierons, and the original value was already out of bounds. Meanwhile fuselages may get heavier from adding new equipment, it was unclear what happened there. The manufacturer proposed to put it into a warm room for a couple of day's and bingo, things changed. Repaired the trailor with Sikaflex and since then its okay.

**Comment by iltis42** [comment]
Wasn't much, less than 0,5%. The water was also in the filler that had been used to equalize repairs.
